src/app.ts

```diff
@@ -41,14 +45,38 @@ app.post('/login', (req, res) => {
 })
 
 const listener = (): void => {
-  const cyan = '\x1b[36m\u001b[1m%s\x1b[0m'
-  const red = '\x1b[31m\u001b[1m%s\x1b[0m'
```

Comment:
why did you remove all these?

Reply:
I did not remove them. Updated my branch and this happened.